File: Algorithms/Java/Exponentiation.java
public class Exponentiation {

	public static void main(String[] args){
		double base = Double.parseDouble(args[0]);
		double exponent = Double.parseDouble(args[1]);
		double result = 1;

		if (exponent > 0)
			for (int iteration = 0; iteration < exponent; iteration++)
				result *= base;
		else if (exponent < 0)
			for (int iteration = 0; iteration > exponent; iteration--)
				result /= base;

		System.out.println(result);
	}

}

File: Algorithms/Python/Exponentiation.py
import sys

def Exponentiation(base, exponent):
	base = float(base)
	exponent = float(exponent)
	result = 1

	intExponent = int(exponent - (exponent % 1))	

	if exponent > 0:
		for iteration in range(0, intExponent):
			result *= base
	elif exponent < 0:
		for iteration in range(0, intExponent):
			result /= base

	print(result)

Exponentiation(sys.argv[1], sys.argv[2])